SERVER/Server.js

```diff
@@ -0,0 +1,153 @@
+const express = require('express');
+const app = express();
+const Constants = require("./util/Constants");
+const {spawn} = require('child_process');
+const fs = require('fs')
+
+//M.init();
+
+const path = require('path');
+
+// JSON via post
+const bodyParser = require('body-parser');
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+// Cookies
+function parseCookies (request) {
+    var list = {},
+        rc = request.headers.cookie;
+
+    rc && rc.split(';').forEach(function( cookie ) {
+        var parts = cookie.split('=');
+        list[parts.shift().trim()] = decodeURI(parts.join('='));
+    });
+
+    return list;
+}
+
+// Static pages
+//app.use(express.static('htmls'));
```

Comment:
You will use this?

Reply:
Currently unsure. Might be useful if we ever develop a web front-end. Using express static would make serving such webpages much easier, but it could be removed for now.